doc/statuscodes.md

```diff
@@ -47,3 +48,5 @@ The following status codes are never generated by the library:
 - ABORTED
 - OUT_OF_RANGE
 - DATA_LOSS
+
+Applications that may wish to retry failed RPCs must decide which status codes on which to retry. As shown in the table above, the gRPC library can generate the same status code for different cases. Server applications can also return those same status codes. Therefore, there is no fixed list of status codes on which it is appropriate to retry in all applications. As a result, individual applications must make their own determination as to which status codes should cause an RPC to be retried.
```

Comment:
Please make the first occurrence of "retry" a link to https://github.com/grpc/proposal/blob/master/A6-client-retries.md.

Reply:
Done!